schema/geoblacklight-schema.json

```diff
@@ -0,0 +1,207 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Schema for GeoBlacklight as implemented for Solr 4.10+. See http://journal.code4lib.org/articles/9710 for more details. Note that the Solr schema uses dynamic typing based on the suffix of the field name. For example, _s denotes a string where _sm denotes a multi-valued string (array of strings).",
+  "id": "http://geoblacklight.org/schema",
+  "title": "GeoBlacklight Schema",
+  "required": [
+    "dc_title_s",
+    "dc_description_s",
+    "dc_rights_s",
+    "dct_provenance_s",
+    "layer_id_s",
+    "layer_geom_type_s",
+    "layer_modified_dt",
+    "layer_slug_s",
+    "solr_geom"
+  ],
+  "type": "array",
+  "properties": {
+    "layer": {
+      "title": "layer",
+      "description": "A GIS data layer. See [this example](https://github.com/OpenGeoMetadata/edu.stanford.purl/blob/master/bb/099/zb/1450/geoblacklight.json) metadata layer.",
+      "type": "object",
+      "properties": {
+        "uuid": {
+          "type": "string",
+          "description": "*DEPRECATED* (use `dc_identifier_s`): Unique identifier for layer that is globally unique.",
+          "example": "http://purl.stanford.edu/vr593vj7147"
+        },
+        "dc_identifier_s": {
+          "type": "string",
+          "description": "Unique identifier for layer as a URI. It should be globally unique across all institutions, assumed not to be end-user visible, and is usually of the form `http://institution/id`. See https://github.com/geoblacklight/geoblacklight/wiki/Schema for more detailed documentation.",
+          "example": "http://purl.stanford.edu/vr593vj7147"
+        },
```

Comment:
Good question. I think it's required (but unused) -- that's how I wrote it up in the doc.